django/db/migrations/state.py

```diff
@@ -87,6 +90,8 @@ def __init__(self, models=None, real_apps=None):
         # Apps to include from main registry, usually unmigrated ones
         self.real_apps = real_apps or []
         self.is_delayed = False
+        # Attributes filled after calling resolve_fields_and_relations
+        self.relations = None
```

Comment:
So that's kind of a reverse relation map similar to `models.Options.related_objects`?

Reply:
Somewhat, yes, but not exactly.
To illustrate, with models like
`app.User` and `app.Author with owner = ForeignKey(app.User)`

`relations` will contain something like
```
{
  ("app", "user"): {
    ("app", "author"): [
      ("owner", ForeignKey)
    ]
  }
}
```
whereas `models.Options.related_objects` contains, for a model, the reverse relation `ManyToOneRel` (and others) objects